brave-lists/debounce.json

```diff
@@ -392,5 +392,34 @@
     ],
     "action": "base64,redirect",
     "param": "a"
+  },
+  {
+    "include": [
+      "https://fmarier.github.io/brave-testing/debouncer/redirect.html?*"
+    ],
+    "exclude": [
+    ],
+    "action": "redirect",
+    "param": "brave_testing"
+  },
+  {
+    "include": [
+      "https://fmarier.github.io/brave-testing/debouncer/redirect.html?*"
+    ],
+    "exclude": [
+    ],
+    "action": "base64,redirect",
+    "param": "brave_testing_base64"
+  },
+  {
+    "include": [
+      "https://fmarier.github.io/brave-testing/debouncer/*"
+    ],
+    "pref": "brave.de_amp.enabled",
```

Comment:
This part doesn't seem correct to me.  We want to allow regex rules in general, independent of the de-amping feature being enabled.  There are just certain specific debouncing rules we want to enable only when the DeAMPing feature is enabled

Reply:
Good point. I copy/pasted that from Shivan's issue, but we don't need it for this.